.github/scripts/check_for_new_version.py

```diff
@@ -66,7 +66,7 @@ def validate_xml(file_name):
     if validate_xml(f"DROID_SignatureFile_V{latest_droid_version}.xml"):
         conf_replaced = (replace_line(line, 'droid', latest_droid_version, "\n") for line in lines)
     if validate_xml(f"container-signature-{latest_containers_version}.xml"):
-        conf_replaced = (replace_line(line, 'containers', latest_containers_version) for line in lines)
+        conf_replaced = (replace_line(line, 'containers', latest_containers_version) for line in conf_replaced)
```

Comment:
Will this return both the `DROID` and `Signature` versions in the `conf_replaced`?

Reply:
Yes, I have tested that locally.